.github/workflows/clang-tidy-modern.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+run-clang-tidy-10.py -config='' -header-filter='^((?!.*kiss_fft\.h.*).)*$,.*' -fix
```

Comment:
Please remove the filter for kiss_fft. Let that come in with the appropriate checks